app/controllers/venues_controller.rb

```diff
@@ -12,6 +12,10 @@ def new
     @venue = Venue.new
   end
 
+  def edit
+    @venue = Venue.find(params[:id])
```

Comment:
These methods should be in the following order in the controller.
index, new, create, edit, update, destroy